client/src/components/RequestForm/RequestForm.tsx

```diff
@@ -0,0 +1,157 @@
+import { Autocomplete, Box, Card, Grid, TextField, Typography } from '@mui/material';
+import { Profile } from '../../interface/Profile';
+import { Star } from '@mui/icons-material';
+import { useStyles } from './useStyles';
+import * as Yup from 'yup';
+import Button from '@mui/material/Button';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Form, Formik, FormikHelpers } from 'formik';
+import { DatePicker, LocalizationProvider } from '@mui/lab';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+
+interface Props {
+  className?: string;
+  profile: Profile;
+  handleSubmit: (
+    {
+      dropInDate,
+      dropInTime,
+      dropOffDate,
+      dropOffTime,
+    }: {
+      dropInDate: Date;
+      dropInTime: Date;
+      dropOffDate: Date;
+      dropOffTime: Date;
+    },
+    {
+      setStatus,
+      setSubmitting,
+    }: FormikHelpers<{
+      dropInDate: Date;
+      dropInTime: Date;
+      dropOffDate: Date;
+      dropOffTime: Date;
+    }>,
+  ) => void;
+}
+
+const getTimes = (hour: number) => {
+  const times = [];
+  for (let i = hour; i < 24; i++) {
+    times.push(`${i % 12 ? i % 12 : 12} ${i / 12 < 1 ? 'am' : 'pm'}`);
+  }
+  return times;
+};
+
+const RequestForm = ({ className, profile, handleSubmit }: Props): JSX.Element => {
+  const classes = useStyles();
+  const today = new Date();
+  console.log(today.toDateString());
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!

Reply:
fix it on PR #70